CRM/Civicase/Setup/CreateSafeFileExtentionOptionValue.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+/**
+ * Class CRM_Civicase_Setup_CreateSafeFileExtentionOptionValue.
+ */
+class CRM_Civicase_Setup_CreateSafeFileExtentionOptionValue {
+
+  /**
+   * Installs 'zip', 'msg', 'eml', 'mbox' Safe File Extention Option values.
+   */
+  public function apply() {
+    $file_extentions_to_be_installed = ['zip', 'msg', 'eml', 'mbox'];
```

Comment:
Let's use camel case for the variable names here for consistency. i.e `$fileExtensionsToBeInstalled`